src/commands/selectModel.ts

```diff
@@ -1,7 +1,7 @@
 import vscode, { ExtensionContext } from "vscode";
 
 import { getExtensionContext } from "../globals/extensionContext";
-import { getModel, getModels } from "../services/codeAssistant";
+import { getServiceApi } from "../services/common";
 import { setDefaultStatus, setLoadingStatus } from "../statusBar/statusBar";
 import { requiresToken } from "../utilities/guards";
```

Comment:
`requiresToken` is not used in the file, the import is not needed

Reply:
thanks for catching.. [removed](https://github.com/Qiskit/qiskit-code-assistant-vscode/pull/102/commits/c496df446c8e73445568667d07acadabb22d9f26)